packages/babel-core/test/api.js

```diff
@@ -754,23 +754,6 @@ describe("api", function() {
       babelrc: false,
     };
 
-    it("only syntax plugin available", function(done) {
```

Comment:
Instead of removing this test, can you replace it with something else? For example, the pipeline operator.